### ISSUE TITLE
ebay-fake-menu-button: when tab through, the focus indicator is missing

### ISSUE THREAD

**Issue by ZZS77** [eBay/ebayui-core#1605]
# Bug Report

## Description
we are using ebay-fake-menu-button component with variations: borderless, no-toggle-icon reverse
Issue:
when tab through, the ebay fake menu button is missing focus indicator e.g. color change highlight blue is missing

## Screenshots
![Screen Shot 2022-01-21 at 2 20 10 PM](https://user-images.githubusercontent.com/42783072/150607818-67eaea87-e846-4ba0-96d2-4e1630d8ca55.png)

**Comment by agliga** [comment]
I believe this usecase is incorrect. We should not be using `borderless` `no-toggle-icon` with an icon inside the menu-button. It expects text. If you have text it correctly underlines the text.
You need to use `variant=overflow` which has focus icon and shows the overflow menu. This is the usecase we support

**Comment by agliga** [comment]
Closing, `variant=overflow` worked
